.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish docs to GitHub Pages
+
+on:
+  workflow_call:
+    inputs:
+      destination_dir:
+        required: true
+        type: string
+    secrets:
+      PUBLISH_DOCS_TOKEN:
```

Comment:
We will also need a `PUBLISH_DOCS_TOKEN` in order to merge this pull request.

Reply:
This has been setup